config/nrfconnect/chip-module/Kconfig

```diff
@@ -18,6 +18,27 @@ rsource "../../zephyr/Kconfig"
 
 if CHIP
 
+config CHIP_WIFI_NRF
```

Comment:
If it's just a composite setting that is used to enable multiple other settings, I think we put such creatures in `Kconfig.features`.

Reply:
Done.